### DISCUSSION TITLE
Load Profile Clustering

### DISCUSSION THREAD

**Discussion by bafs84** [TDAmeritrade/stumpy#1037] [Help: Best Practices]
I'm looking at an anonymized [smart meter dataset from Switzerland](https://open.data.axpo.com/%24web/index.html#dataset-a). Each file contains a full month in 15 minute resoltion of about 200'000 meters. There are several years of data, in total about 900GB of CSV files.

Reading the original papers, I had hope to be able to use Matrix Profiles to do:
- identify common pattern/motifs that are shared by _some_ timeseries (like [shapelets](https://stumpy.readthedocs.io/en/latest/Tutorial_Shapelet_Discovery.html) but i have no labels)
- cluster timeseries according to some kind of MP_dist or 
- understand trends or developments within these clusters (maybe using chains?)

This question might be related to the [MP XXVII paper](https://www.dropbox.com/sh/qtmb8uxqntpt1e4/AAAI8dMmL4IQ2AUsx_6q0TsWa/PRCIS_ICKG_Extended.pdf), but I'd prefer stumpy - and it's ecosystem with support for dask and GPU. 

Could you advise how to go forward?

**Comment by JaKasb** [comment]
The following publications should help you:
https://www.mdpi.com/1424-8220/21/23/8036
https://link.springer.com/chapter/10.1007/978-981-16-2377-6_5

Disclaimer: I am an author in these publications.

**Reply by bafs84** [reply]
Dear Jakob 
Thank you for the ideas and fast response.  I see some differences that make it difficult for me to understand how to go about it:
- your sampling rate is higher (per second resolution), compared to 15 minute. this slow sampling is done on purpose to protect the privacy of customers to avoid disaggregation. so I will not be able to cluster smart meters by devices behind the meter.
- I have no labels in my data
